pkgs/by-name/ja/jawiki-all-titles-in-ns0/package.nix

```diff
@@ -0,0 +1,37 @@
+{ lib
+, fetchurl
+, stdenvNoCC
+}:
+
+let
+date = "20240520";
+jawiki = fetchurl {
+  url ="https://dumps.wikimedia.org/jawiki/${date}/jawiki-${date}-all-titles-in-ns0.gz";
+  hash= "sha256-HMQutYuKiI7fXmbivS0mIgdHsQu6t6WdVFVrnyw8JIw=";
+};
+in stdenvNoCC.mkDerivation {
+  pname = "jawiki-all-titles-in-ns0";
+  version = date;
+
+  unpackPhase = "true";
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir $out
+    cp ${jawiki} $out/jawiki-all-titles-in-ns0.gz
+
+    runHook postInstall
+  '';
+
+  meta = with lib;{
```

Comment:
[Avoid `with lib;`](https://github.com/NixOS/nixpkgs/issues/208242) 

Same applies for other files but I won't dwell too much on this out of brevity